python/activity_stream/label_widgets.py

```diff
@@ -0,0 +1,206 @@
+# Copyright (c) 2015 Shotgun Software Inc.
+# 
+# CONFIDENTIAL AND PROPRIETARY
+# 
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit 
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your 
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights 
+# not expressly granted therein are reserved by Shotgun Software Inc.
+"""
+A collection of various subclassed QLabels used by the activity stream UI
+"""
+from sgtk.platform.qt import QtCore, QtGui
+from . import utils
+import sgtk
+
+class ClickableTextLabel(QtGui.QLabel):
```

Comment:
if i remember correctly, i was having issues getting the text correctly centered and went for a solution that i knew would work. But i am sure it's possible to get it to work - and more elegant too...